src/ripples/circle-ripple.jsx

```diff
@@ -25,11 +35,29 @@ const CircleRipple = React.createClass({
 
   getDefaultProps() {
     return {
-      color: Colors.darkBlack,
       opacity: 0.16,
     };
   },
 
+  getInitialState() {
```

Comment:
Can you move this logic to `TouchRipple`? It would be better to keep this component stateless.